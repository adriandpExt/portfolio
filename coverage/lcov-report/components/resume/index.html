
<!doctype html>
<html lang="en">

<head>
    <title>Code coverage report for components/resume</title>
    <meta charset="utf-8" />
    <link rel="stylesheet" href="../../prettify.css" />
    <link rel="stylesheet" href="../../base.css" />
    <link rel="shortcut icon" type="image/x-icon" href="../../favicon.png" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <style type='text/css'>
        .coverage-summary .sorter {
            background-image: url(../../sort-arrow-sprite.png);
        }
    </style>
</head>
    
<body>
<div class='wrapper'>
    <div class='pad1'>
        <h1><a href="../../index.html">All files</a> components/resume</h1>
        <div class='clearfix'>
            
            <div class='fl pad1y space-right2'>
                <span class="strong">68.29% </span>
                <span class="quiet">Statements</span>
                <span class='fraction'>28/41</span>
            </div>
        
            
            <div class='fl pad1y space-right2'>
                <span class="strong">100% </span>
                <span class="quiet">Branches</span>
                <span class='fraction'>0/0</span>
            </div>
        
            
            <div class='fl pad1y space-right2'>
                <span class="strong">0% </span>
                <span class="quiet">Functions</span>
                <span class='fraction'>0/7</span>
            </div>
        
            
            <div class='fl pad1y space-right2'>
                <span class="strong">66.66% </span>
                <span class="quiet">Lines</span>
                <span class='fraction'>26/39</span>
            </div>
        
            
        </div>
        <p class="quiet">
            Press <em>n</em> or <em>j</em> to go to the next uncovered block, <em>b</em>, <em>p</em> or <em>k</em> for the previous block.
        </p>
        <template id="filterTemplate">
            <div class="quiet">
                Filter:
                <input type="search" id="fileSearch">
            </div>
        </template>
    </div>
    <div class='status-line medium'></div>
    <div class="pad1">
<table class="coverage-summary">
<thead>
<tr>
   <th data-col="file" data-fmt="html" data-html="true" class="file">File</th>
   <th data-col="pic" data-type="number" data-fmt="html" data-html="true" class="pic"></th>
   <th data-col="statements" data-type="number" data-fmt="pct" class="pct">Statements</th>
   <th data-col="statements_raw" data-type="number" data-fmt="html" class="abs"></th>
   <th data-col="branches" data-type="number" data-fmt="pct" class="pct">Branches</th>
   <th data-col="branches_raw" data-type="number" data-fmt="html" class="abs"></th>
   <th data-col="functions" data-type="number" data-fmt="pct" class="pct">Functions</th>
   <th data-col="functions_raw" data-type="number" data-fmt="html" class="abs"></th>
   <th data-col="lines" data-type="number" data-fmt="pct" class="pct">Lines</th>
   <th data-col="lines_raw" data-type="number" data-fmt="html" class="abs"></th>
</tr>
</thead>
<tbody><tr>
	<td class="file medium" data-value="Resume.tsx"><a href="Resume.tsx.html">Resume.tsx</a></td>
	<td data-value="75" class="pic medium">
	<div class="chart"><div class="cover-fill" style="width: 75%"></div><div class="cover-empty" style="width: 25%"></div></div>
	</td>
	<td data-value="75" class="pct medium">75%</td>
	<td data-value="16" class="abs medium">12/16</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="0" class="abs high">0/0</td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="3" class="abs low">0/3</td>
	<td data-value="73.33" class="pct medium">73.33%</td>
	<td data-value="15" class="abs medium">11/15</td>
	</tr>

<tr>
	<td class="file medium" data-value="TimelineCard.tsx"><a href="TimelineCard.tsx.html">TimelineCard.tsx</a></td>
	<td data-value="59.09" class="pic medium">
	<div class="chart"><div class="cover-fill" style="width: 59%"></div><div class="cover-empty" style="width: 41%"></div></div>
	</td>
	<td data-value="59.09" class="pct medium">59.09%</td>
	<td data-value="22" class="abs medium">13/22</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="0" class="abs high">0/0</td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="4" class="abs low">0/4</td>
	<td data-value="57.14" class="pct medium">57.14%</td>
	<td data-value="21" class="abs medium">12/21</td>
	</tr>

<tr>
	<td class="file high" data-value="index.ts"><a href="index.ts.html">index.ts</a></td>
	<td data-value="100" class="pic high">
	<div class="chart"><div class="cover-fill cover-full" style="width: 100%"></div><div class="cover-empty" style="width: 0%"></div></div>
	</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="2" class="abs high">2/2</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="0" class="abs high">0/0</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="0" class="abs high">0/0</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="2" class="abs high">2/2</td>
	</tr>

<tr>
	<td class="file high" data-value="utils.tsx"><a href="utils.tsx.html">utils.tsx</a></td>
	<td data-value="100" class="pic high">
	<div class="chart"><div class="cover-fill cover-full" style="width: 100%"></div><div class="cover-empty" style="width: 0%"></div></div>
	</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="1" class="abs high">1/1</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="0" class="abs high">0/0</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="0" class="abs high">0/0</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="1" class="abs high">1/1</td>
	</tr>

</tbody>
</table>
</div>
                <div class='push'></div><!-- for sticky footer -->
            </div><!-- /wrapper -->
            <div class='footer quiet pad2 space-top1 center small'>
                Code coverage generated by
                <a href="https://istanbul.js.org/" target="_blank" rel="noopener noreferrer">istanbul</a>
                at 2024-06-22T07:57:52.871Z
            </div>
        <script src="../../prettify.js"></script>
        <script>
            window.onload = function () {
                prettyPrint();
            };
        </script>
        <script src="../../sorter.js"></script>
        <script src="../../block-navigation.js"></script>
    </body>
</html>
    