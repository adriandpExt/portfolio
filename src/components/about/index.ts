export * from "./About";
export * from "./Wrapper";
