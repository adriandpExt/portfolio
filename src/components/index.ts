export * from "./appbar";
export * from "./svg-icons";
export * from "./aside";
