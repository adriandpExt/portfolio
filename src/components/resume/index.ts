export * from "./Resume";
export * from "./TimelineCard";
