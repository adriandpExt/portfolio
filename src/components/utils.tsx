export const toggleDarkTheme = (isDarkMode: boolean) => {
  return isDarkMode ? "bg-gray-900 text-white" : "bg-gray-200 text-gray-800";
};

export const toggleLightTheme = (isDarkMode: boolean) => {
  return isDarkMode ? "bg-gray-200 text-gray-800" : "bg-gray-900 text-white";
};

export const handleToggleDropIcon = (isDarkMode: boolean) => {
  return isDarkMode ? "bg-gray-200  text-gray-800" : "bg-gray-900  text-white";
};
