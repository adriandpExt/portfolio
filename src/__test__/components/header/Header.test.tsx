import "@testing-library/jest-dom";
import { render } from "@testing-library/react";
import { Header } from "../../../components";

const mockedNavigate = jest.fn();

jest.mock("react-router-dom", () => ({
  ...jest.requireActual("react-router-dom"),
  useNavigate: () => mockedNavigate,
}));

describe("header component test", () => {
  test("should render header", () => {
    const { container } = render(<Header />);

    expect(container).toMatchSnapshot();
  });

  test("should render name", () => {
    const { getByTestId } = render(<Header />);

    expect(getByTestId("fullname")).toBeInTheDocument();
  });

  test("should render profession", () => {
    const { getByTestId } = render(<Header />);

    expect(getByTestId("profession")).toBeInTheDocument();
  });
});
