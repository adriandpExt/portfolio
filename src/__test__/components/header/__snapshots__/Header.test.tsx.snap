// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`header component test should render header 1`] = `
<div>
  <div
    class="MuiBox-root css-1nhtd43"
  >
    <h2
      class="MuiTypography-root MuiTypography-h2 css-o3dvd8-MuiTypography-root"
      data-testid="fullname"
    >
      <span
        data-testid="react-typed"
      >
        <span />
        <span
          aria-hidden="true"
          class="typed-cursor"
        >
          |
        </span>
      </span>
    </h2>
    <h5
      class="MuiTypography-root MuiTypography-h5 css-kcegrn-MuiTypography-root"
      data-testid="profession"
    >
      I am a
      <span
        style="margin: 0px 0.5rem; font-weight: 600; background: rgb(255, 99, 71); font-family: monospace; font-size: 2rem;"
      >
        <span
          data-testid="react-typed"
        >
          <span />
          <span
            aria-hidden="true"
            class="typed-cursor"
          >
            |
          </span>
        </span>
      </span>
    </h5>
    <button
      class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeLarge MuiButton-containedSizeLarge MuiButton-colorPrimary MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeLarge MuiButton-containedSizeLarge MuiButton-colorPrimary css-qnifjw-MuiButtonBase-root-MuiButton-root"
      tabindex="0"
      type="button"
    >
      see projects
      <span
        class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
      />
    </button>
  </div>
</div>
`;
